registration-processor-default.properties

```diff
@@ -478,13 +478,22 @@ mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.format.url.mock-1.1
 mosip.registration.processor.credentialtype=euin
 mosip.registration.processor.encrypt=false
 mosip.registration.processor.credential.request.service.id=mosip.credential.request.generator 
-mosip.registration.processor.digitalcard.credentialtype=PDFCard
-mosip.registration.processor.print.credentialtype=euin
-mosip.registration.processor.digitalcard.template=RPR_UIN_CARD_TEMPLATE
-mosip.registration.processor.print.template=RPR_UIN_CARD_TEMPLATE
-
-# this property is used to define issuers eg partnerid:credentialType:templateTypeCode
-mosip.registration.processor.issuer=mpartner-default-print:${mosip.registration.processor.print.credentialtype}:${mosip.registration.processor.print.template};mpartner-default-digitalcard:${mosip.registration.processor.digitalcard.credentialtype}:${mosip.registration.processor.digitalcard.template}
+# This property is used to define default internal credential issuers separated by comma.
+# eg: partnerid#credentialType#templateTypeCode
+mosip.registration.processor.default.internal.issuers=mpartner-default-digitalcard#PDFCard#RPR_UIN_CARD_TEMPLATE
+# This property is used to define default issuers separated by comma,
+# eg: partnerid#credentialType#templateTypeCode
+mosip.registration.processor.default.issuers=mpartner-default-print#euin#RPR_UIN_CARD_TEMPLATE
+
+# Credential Partner Configuration
+# First language defined in the mandatory language list will be considered to identify the attribute value.
+mosip.mandatory-languages=eng
+# Map all print partners for a respective attribute ('state') value (any appropriate valid id-schema field) in the format given below,
+# key and value pair where key would be string value (partnerId#credential_type#template_code) and value would be an expression
+# ({list_of_attribute_values} contains attribute_field)
+mosip.registration.processor.print.issuer.config-map={'mpartner-msp-print#vercred#RPR_UIN_CARD_TEMPLATE':'{"Karnataka", "Tamil Nadu"} contains ${mosip.identification.attribute}', 'mpartner-pdf-print#vercred#RPR_UIN_CARD_TEMPLATE': '{"Andra Pradesh"} contains ${mosip.identification.attribute}'}
```

Comment:
The reference of mosip.identification.attribute to be changed to state

Reply:
Also state is not present in https://github.com/mosip/mosip-config/blob/develop-v3/identity-mapping.json so we have to use something which is already available in the above sample config

Reply:
Changed the 'state' field to 'region' and 
mosip.identification.attribute=region, so that the reference can be used for multiple partners inside the expression.